packages/drupal/custom/src/ContentHub.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Drupal\custom;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\graphql_directives\DirectiveArguments;
+
+/**
+ * Service to query content listings.
+ */
+class ContentHub {
+
+  /**
+   * The entity type manager, to query and load pages.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected EntityTypeManagerInterface $entityTypeManager;
```

Comment:
Another idea: https://stitcher.io/blog/constructor-promotion-in-php-8

Reply:
I actually had this, but it was weird to have an empty constructor :D probably i'm old.